lib/outbound.php

```diff
@@ -185,13 +190,11 @@ private static function _build_user($user_id, Array $user_info=null, Array $user
                 'group_id' => null,
                 'group_attributes' => null,
                 'previous_id' => null,
+                'attributes' => null,
             )
         );
         $user_info['user_id'] = $user_id;
-
-        if ($user_attrs) {
-            $user_info['attributes'] = $user_attrs;
-        }
+        
```

Comment:
Do a [filter](http://php.net/manual/en/function.array-filter.php) over `$user_info` and remove empty values.

    return array_filter($user_info, function($v) {return !empty($v);});